docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -196,7 +196,7 @@ If there is an unreleased feature you would like to try, it may be available in
   <dependency>
     <groupId>software.amazon.jdbc</groupId>
     <artifactId>aws-advanced-jdbc-wrapper</artifactId>
-    <version>2.3.2-SNAPSHOT</version>
+    <version>2.3.3-SNAPSHOT</version>
```

Comment:
The snapshot version should be ahead of the current, so 2.3.4 in this case I believe

Reply:
Good catch!

Reply:
Fixed